app/views/ontologies/_form.html.haml

```diff
@@ -136,85 +19,199 @@
-        %span.asterik *
-      %div.col-md-11
-        = f.select(:administeredBy, @user_select_list ,{selected: @ontology.administeredBy || session[:user].id}, {multiple: true, :"data-placeholder" => "Select administrators", class:"form-control"})
-    %div.form-group.row
-      %div.col-md-1
-        = f.label :hasDomain, "Categories", class:"col-form-label"
-      %div.col-md-11
-        - cat_select = @categories.sort{|a,b| a.name <=> b.name}.map{|c| [c.name, c.id]}
-        = f.hidden_field(:hasDomain, {value: "", id: "ontology_hasDomain_empty_select_hack", name: "ontology[hasDomain][]"})
-        = f.select(:hasDomain, cat_select, {selected: @ontology.hasDomain}, {multiple: true, :"data-placeholder" => "Select category (domain)", class:"form-control"})
-    %div.form-group.row
-      %div.col-md-1
-        = f.label :group, "Groups", class:"col-form-label"
-      %div.col-md-11
-        - group_select = @groups.sort{|a,b| a.name <=> b.name}.map{|c| [c.name, c.id]}
-        = f.hidden_field(:group, {value: "", id: "ontology_group_empty_select_hack", name: "ontology[group][]"})
-        = f.select(:group, group_select, {selected: @ontology.group}, {multiple: true, :"data-placeholder" => "Select group", class:"form-control"})
-    %div.form-row
-      %div.from-group.col-md-6
-        = f.label :isView, "This ontology is a view of:"
-        = f.check_box :isView, checked: @ontology.view?
-        %div#viewOf_picker.row.form-group
-          - single_picker_locals = {:picker_id => "ontology_viewOf", placeholder: "Select an ontology to create a view on", field_name: "viewOf", disabled: !@ontology.view?, selected: @ontology.viewOf}
+.upload-ontology-container
+  .upload-ontology-card
+    .upload-ontology-center
+      .Upload-ontology-title
+        %div Upload ontology
+        %hr
+    .upload-ontology-progress
+      .details.progress-item
+        .outlined-circle.outlined-active-circle
+          %img{src: asset_path("white-check.svg")}
+        .active 
+          Details
+      %hr.line
+      .General-metadata.progress-item
+        .outlined-circle
+          %img{src: asset_path("white-check.svg")}
+        %div General metadata
+      %hr.line
+      .Dates-and-contacts.progress-item
+        .outlined-circle
+          %img{src: asset_path("white-check.svg")}
+        %div Dates  contacts
+    
+    
+    
+    .upload-ontology-details-step{"data-form": "1"}
+      .upload-ontology-input-field-container
+        = render Form::TextInputComponent.new(label: "Name", name: "ontology[name]")
+
+      .upload-ontology-input-field-container
+        = render Form::TextInputComponent.new(label: "Acronym", name: "ontology[acronym]")
+      .upload-ontology-field-container
+        = render Form::SelectComponent.new(label: "Administrator", name: "ontology[administeredBy]", value: @user_select_list, selected: @ontology.administeredBy || session[:user].id, multiple: true, helper_text: "You should select one admin at least")
```

Comment:
here you removed two inputs that need to put again: 

-  Public or Private 
<img width="336" alt="image" src="https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/9b339b5a-5da0-411a-b8a2-672da42d3386">

- And when private give access to a list of users 
 
<img width="1371" alt="image" src="https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/dd0b1428-0ac8-401e-aab4-5ddc4dda1c17">